server/wizard/steps/step_webhook_events.go

```diff
@@ -0,0 +1,43 @@
+package steps
+
+import (
+	"fmt"
+
+	"github.com/mattermost/mattermost-plugin-api/experimental/flow/steps"
+)
+
+const (
+	stepNameWebhookEvents = "webhook_events"
+
+	stepTitleWebhookEvents = "Select webhook events"
+
+	stepDescriptionWebhookEvents = `Click **Add events** and select the following events:
+
+- Meeting
+	- End Meeting
+	- Participant/Host joined meeting
+	- Participant/Host left meeting
+- Recording
+	- All recordings have completed
```

Comment:
We can use these in future features, and I'm wondering if we should ask admins to update the events later when we support a feature for them, or if we should have them preemptively set them to make it an easier transition when we support the event types.

Reply:
I consider that we should ask only for what we use.
I understand the feeling, but I think just asking for what is needed will force us to consider the update path in future releases, which is a good thing.

Reply:
[We have discussed](https://community-daily.mattermost.com/core/pl/51mu7o7k9tnnjpbhwag9oaw48o) how we can tell admins of new features in plugin releases, though I don't think we've landed on a proper solution to implement. Maybe we can revive that discussion. @aaronrothschild What do you think?

I'm fine with removing these as it is incorrect to have them at the moment.